### PR TITLE
Added team schedule and matchup scores functionality 

### PR DESCRIPTION
**Pull request by ZachMyers3**
I've done the following:

1. Added a schedule function that fetches the schedule for a given team name. Much like the team call but instead returns the schedule with the scoreID for each of these matches. On top of this I've added and example to the readme and added a test spec file.

2. Added a scores function that returns the scores for the home and away team for each player in the matchup given a leagueId, scoreId, and season number. Example added to the readme and a test spec file added for this as well. 

I don't know how much you were interested in extending the functionality but I figured I'd throw in this request to see if you were interested. Feel free to comment.

### CODE REVIEW

**Comment by BenMagyar** [comment]
That's quite large 😮. I'll try and look at it this weekend and review.

**Comment by ZachMyers3** [comment]
Did you ever get a chance to look at the changes? Thanks.

File: package.json [hunk 1]
```diff
@@ -1,15 +1,72 @@
 {
-  "name": "flea-killer",
-  "version": "0.1.5",
+  "_args": [
```
Comment (BenMagyar) [inline package.json:2]: Ton of package.json changes?

File: README.md [hunk 2]
```diff
@@ -141,3 +141,80 @@ flea.draft.get('mlb', 13157, 2014, function(err, draft){
...
+```
+{
+  id: number,
+  home_name: string,  
+  away_name: string,
+  home_scores: [{
+    player: string,
+    score: number,
+    position: string
+    }, ...],
```
Comment (BenMagyar) [inline README.md:209]: Formatting here is off a bit.

File: README.md [hunk 3]
```diff
@@ -141,3 +141,80 @@ flea.draft.get('mlb', 13157, 2014, function(err, draft){
...
+  away_scores: [{
+    player: string,
+    score: number,
+    position: string
+    }, ...]
```
Comment (BenMagyar) [inline README.md:214]: Formatting here is off a bit.

File: .npmignore [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Logs
```
Comment (BenMagyar) [inline .npmignore:1]: Rather just use files in `package.json` if there is a reason to exclude files.

**Comment by BenMagyar** [comment]
@ZachMyers3, sorry about that! Lost track of it. Left a few comments. Thanks!

**Comment by ZachMyers3** [comment]
I'll try and take a look at these tomorrow if I have time, thanks.

**Pull request closed without merging**